code/game/objects/effects/holy.dm

```diff
@@ -0,0 +1,36 @@
+//Effects used for anything holy and divine currently used for smite and spawming a admin dummy... maybe for holy explosion one day
+/obj/effect/holy
+	name = "holy"
+	icon = null
+	icon_state = null
+	pixel_y = 0
+	pixel_x = 0
+	mouse_opacity = 0
+
+	proc/start(atom/location)
+		loc = get_turf(location)
+		spawn(20)
+			qdel(src)
+
+/obj/effect/holy/lightning
+	name = "divine retribution"
+	icon = 'icons/effects/224x224.dmi'
```

Comment:
Yes, i fully agree with @JamieH wanting to put all sprites under their own folder, just create a new `dmi` for this specific effect or a generic `effects.dmi` in the hippie sprite folder so whenever we rebase in the future it'll be faster and more efficient